### PR TITLE
MakerDAO and Compound Tests

### PR DESCRIPTION
**Pull request by kendricktan**

### CODE REVIEW

File: packages/money-legos/maker.ts [hunk 1]
```diff
@@ -1,70 +1,77 @@
...
-import networkIds from "./networks"
+import networkIds from "./networks";
 
 import makerProxyRegistryAbi from "./abi/makerdao/ProxyRegistry.json";
 import dssCdpManagerAbi from "./abi/makerdao/DssCdpManager.json";
 import dssProxyActionsAbi from "./abi/makerdao/DssProxyActions.json";
 
 const ilks = {
-  "BAT-A": {
+  bat_a: {
```
Comment (adrianmcli) [inline packages/money-legos/maker.ts]: how come underscore? why not `ethA`? What is the `A` for anyway?

File: packages/money-legos/maker.ts [hunk 2]
```diff
@@ -1,70 +1,77 @@
...
     join: {
       address: {
         [networkIds.mainnet]: "0x3D0B1912B66114d4096F48A8CEe3A56C231772cA"
-      },
+      }
     }
   },
-  "ETH-A": {
+  eth_a: {
+    ilk: "ETH-A",
```
Comment (adrianmcli) [inline packages/money-legos/maker.ts]: I think this should be called `symbol`. Ilk is just [defined](https://github.com/makerdao/dss/wiki/Glossary#cdp-engine---vat) as a "collateral type". The string "ETH-A" is like a ticker symbol on a stock market. At least that's my understanding.

```js
// this doesn't really make sense, there are 2 ilks in that sentence
maker.ilks.eth_a.ilk

// this makes more sense
maker.ilks.eth_a.symbol
```

File: packages/money-legos/maker.ts [hunk 3]
```diff
@@ -1,70 +1,77 @@
...
     token: {
       address: Object.keys(networkIds).reduce((acc, x) => {
         // @ts-ignore
         acc[x] = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```
Comment (adrianmcli) [inline packages/money-legos/maker.ts:24]: This is under `maker.ts` but I thought this was `aave` specific? I'm not sure.

File: packages/money-legos/uniswap.ts [hunk 4]
```diff
@@ -4,13 +4,13 @@ import uniswapFactoryAbi from "./abi/uniswap/Factory.json";
 import uniswapExchangeAbi from "./abi/uniswap/Exchange.json";
 
 const uniswap = {
-  uniswapFactory: {
+  factory: {
```
Comment (adrianmcli) [inline packages/money-legos/uniswap.ts:7]: niiiiiceee, makes a lot of sense. `uniswap.uniswapFactory` is so redundant lol

File: packages/smart-contracts/.mocharc.json [hunk 5]
```diff
@@ -2,5 +2,5 @@
     "extension": ["ts"],
     "spec": "test/**/*.ts",
     "require": "ts-node/register",
-    "timeout": 120000
+    "timeout": 1200000
```
Comment (adrianmcli) [inline packages/smart-contracts/.mocharc.json:5]: btw, why Mocha and not Jest?

**Review by adrianmcli — CHANGES_REQUESTED** [review]
A few questions and comments

File: packages/money-legos/maker.ts (hunk 1 above)
Comment (kendricktan) [reply packages/money-legos/maker.ts]: Im just following the makerdao std https://changelog.makerdao.com/releases/mainnet/1.0.4/contracts.json

File: packages/money-legos/maker.ts (hunk 2 above)
Comment (kendricktan) [reply packages/money-legos/maker.ts]: Sure

File: packages/money-legos/maker.ts (hunk 3 above)
Comment (kendricktan) [reply packages/money-legos/maker.ts:24]: Thats true we can just leave it as null

File: packages/smart-contracts/.mocharc.json (hunk 5 above)
Comment (kendricktan) [reply packages/smart-contracts/.mocharc.json:5]: I was using `ethereum-waffle` for testing and that was using mocha so i continued using it. If you wanma change it to jest feel free to do so.

File: packages/money-legos/maker.ts (hunk 1 above)
Comment (adrianmcli) [reply packages/money-legos/maker.ts]: Those are constants exposed via a JSON API. For Javascript libraries, camelCase is always the convention. The only exception is if they are global constants (like the json file you linked), then you can use all-caps with underscores. Lowercase with underscore is almost never used in modern JS.

Source: https://stackoverflow.com/questions/921133/javascript-naming-conventions

**Comment by kendricktan** [comment]
Added compound tests

File: packages/smart-contracts/.mocharc.json (hunk 5 above)
Comment (adrianmcli) [reply packages/smart-contracts/.mocharc.json:5]: yea maybe later

**Comment by adrianmcli** [comment]
lgtm

**Pull request merged**